ios/FluentUI/List/ListItem.swift

```diff
@@ -39,11 +40,11 @@ public struct ListItem<LeadingContent: View,
         self.leadingContent = leadingContent
         self.trailingContent = trailingContent
         self.action = action
-        let layoutType = ListItem.layoutType(subtitle: subtitle, footer: footer)
-        self.tokenSet = ListItemTokenSet(customViewSize: { layoutType.leadingContentSize })
     }
 
     public var body: some View {
+        let tokenSet = ListItemTokenSet(customViewSize: { leadingContentSize })
+        tokenSet.replaceAllOverrides(with: tokenOverrides)
```

Comment:
Out of curiosity, what's the perf impact observed or expected here? We can't guarantee when body runs so hopefully can guarantee this is dirt cheap?

Reply:
We talked about this offline quite a bit, and yep, token set creation is dirt cheap. We'll work in the future to make it even cheaper, but performance should be totally acceptable right now.